.kitchen.yml

```diff
@@ -0,0 +1,24 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  attributes:
+    authorization:
+      sudo:
+        users:
+          - vagrant
+
+platforms:
+- name: windows-2012r2
+  transport:
+    name: winrm
+    elevated: true
+    elevated_username: System
+    elevated_password: null
+  driver_config:
+    box: criteo-windows-2012r2-standard
```

Comment:
this box is not accessible from the internet.

I suggest not to commit this kitchen.yml file since it will be hard to use over kitchen.ec2